instance/models/instance.py

```diff
@@ -52,14 +52,25 @@ class InstanceReference(TimeStampedModel):
     """
     InstanceReference: Holds common fields and provides a list of all Instances
 
-    Has name, created, and modified fields for each Instance.
+    Has name, created, instance_type_id, and modified fields for each Instance.
 
     Instance is an abstract class, so having this common InstanceReference class gives us a
     fully generic way to iterate through all instances and allow instances to be implemented
     using a variety of different python classes and database tables.
     """
+    INSTITUTIONAL = "0"
+    PROTEACHER = "1"
+    SANDBOX = "2"
+    DEVELOPMENT = "3"
+
     name = models.CharField(max_length=250, blank=False, default='Instance')
     instance_type = models.ForeignKey(ContentType, on_delete=models.CASCADE)
+    instance_purpose = models.CharField(max_length=20,
```

Comment:
I wasn't sure what to name this variable, turns out we already have `instance_type` property. I'm fully open to renaming this to something more descriptive.